main.php

```diff
@@ -71,6 +70,21 @@
 		exit(1);
 	}
 }
+// Allow a first class env var to be set that disables .env file loading
+if (!getenv('SS_IGNORE_DOT_ENV')) {
```

Comment:
Disagree. We want .env files to be loaded by default, and I strongly disagree that `SS_USE_DOT_ENV = null` will have the opposite behaviour of `SS_USE_DOT_ENV = false`. That's way ickier.